.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+name: "Tests"
+
+on: [pull_request, push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env: 
+      CIVICRM_VERSION: 5.9.beta1
+    
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Build CiviCRM for testing
+      run: | 
+        mkdir build
+        git clone -b $CIVICRM_VERSION --depth 1 https://github.com/compucorp/civicrm-core.git build/civicrm-core
```

Comment:
Is the build folder really necessary?

Reply:
Not necessary, only did this for demonstrate the path agreement.